pyirf/scripts/lst_performance.py

```diff
@@ -157,7 +156,6 @@ def main(args):
                 continue
 
             psf = np.percentile(data['offset'], radius)
-            psf_err = psf / np.sqrt(len(data))
```

Comment:
Indeed this is not used, but I think we should keep it somewhere (I don't remember, is it defined in OGADF?)

Reply:
The formula is also incorrect for estimating an uncertainty for the psf.